packages/ui-components/src/utils/responsive-utils/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './break-points.util';
+export { default as useBreakpoint } from './use-breakpoint';
```

Comment:
After changed the hook to named export you can just import here without the use of aliasing